src/components/ForgotPassword/ForgotPassword.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from "react";
+import axios from "axios";
```

Comment:
Please, don't use `axios`. You need to use only `fetch` according to the criteria.